### PR TITLE
Add Bolivia national shields

### PR DESCRIPTION
**Pull request by whubsch**
This pull requests adds road shields for Bolivia's national route network. I can't find any evidence that these shields are used on the ground, but they are used in cartographic representations of the road network. National legislation also outlines designs for departmental and municipal route shields (see below), but I can't find any evidence that those are used in any context.

I've chosen just to reuse the green US interstate business route shield for simplicity's sake.

Screenshot 2024-10-15 at 5 55 50 AM

[localhost link](http://localhost:1776/#map=10.47/-17.9538/-67.0857)
Screenshot 2024-11-26 at 6 10 18 AM

[departmental map of Oruro](https://www.abc.gob.bo/wp-content/uploads/2018/08/MAPA_ORURO_2018.pdf)
Screenshot 2024-11-26 at 6 10 29 AM

### CODE REVIEW

**Comment by quincylvania** [comment]
It seems like the US route shield would be a better fit than the interstate shield? There is a function to set the color without adding another sprite.

**Review by ZeLonewolf — CHANGES_REQUESTED** [review]
Thanks for the PR.  I agree that a badge shield (like the ones used in the US Numbered Highway System) is the right style for this network.

File: src/js/shield_defs.js [hunk 1]
```diff
@@ -670,6 +670,7 @@ export function loadShields() {
   shields["US:I:Business:Spur"] = shields["US:I:Business:Loop"];
   shields["US:I:Downtown:Loop"] = shields["US:I:Business:Loop"];
   shields["US:I:Downtown:Spur"] = shields["US:I:Business:Spur"];
+  shields["BO:fundamental"] = shields["US:I:Business:Loop"];
```
[the remark this answers is not in the document]
Comment (whubsch) [reply src/js/shield_defs.js]: Handy, didn't realize that was a possibility. Latest commit should use the generic `badgeShieldCrossbar` icon:

Screenshot 2024-11-30 at 5 17 15 PM

**Comment by whubsch** [comment]
Thanks for the suggestions! The latest commit should use the generic `badgeShieldCrossbar` icon.

**Review by ZeLonewolf — APPROVED** [review]
Looks great!  Thanks for the contribution.

**Pull request merged**